recipes/pyrat/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyratlib" %}
+{% set version = "0.5.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pyratlib/pyratlib-0.5.2.tar.gz
+  sha256: 0232710ba0b3da698a0732cf5552c95a6dd213c5a48e65aa8d41a82e6b0857be
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python-neo
+    - matplotlib-base
+    - numpy
+    - pandas
+    - python >=3.7
+    - scikit-learn
+
+test:
+  imports:
+    - pyratlib
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/pyratlib/pyrat
+  summary: PyRat is a user friendly library in python to analyze data from the DeepLabCut. Developed to help researchers unfamiliar with programming can perform animal behavior analysis more simpler.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - pyratlib
+    - tuliofalmeida
+    - abnr

```

Comment:
@tuliofalmeida @abnr Are you happy to be listed as maintainers here?

Reply:
Yes @chrisburr. I totally agree.

Reply:
Yes @chrisburr

Reply:
Hi Chris, everything OK with PyRAT?